src/main/java/com/linkedin/kmf/apps/MultiClusterMonitor.java

```diff
@@ -29,17 +30,18 @@
 public class MultiClusterMonitor implements App {
   private static final Logger LOG = LoggerFactory.getLogger(MultiClusterMonitor.class);
 
-  private final MultiClusterTopicManagementService _topicManagementService;
+  private final MultiClusterTopicManagementService _multiClusterTopicManagementService;
   private final ProduceService _produceService;
   private final ConsumeService _consumeService;
   private final String _name;
 
   public MultiClusterMonitor(Map<String, Object> props, String name) throws Exception {
```

Comment:
I like to avoid kicking off async operations in the constructor. The main reason is that constructors can't return anything. I would strongly encourage you to separate construction from kicking off the async operation so that the future can be returned.

Reply:
Asynchronous operations will be refactored out of constructor in a separate PR.